### PR TITLE
📄 Add dependency license info to image layer

### PR DESCRIPTION
**Pull request by grdryn**

### CODE REVIEW

File: licenses/rpm-licenses.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                    <url>https://raw.githubusercontent.com/memcached/memcached/1.4.15/LICENSE</url>
+                </license>
+            </licenses>
+        </dependency>
+        <dependency>
+            <packageName>nmap-ncat</packageName>
+            <version>6.40</version>
+            <licenses>
+                <license>
+                    <name>GPLv2 with modifications</name>
```
Comment (grdryn) [inline licenses/rpm-licenses.xml]: I'm not sure if this is the correct way to phrase this, or even if it's a correct approximation. In our Dockerfile, we install `nmap-ncat` (that's the rpm name, even though we ask for `nc`). The License field of the RPM metadata says `GPLv2 and LGPLv2+ and GPLv2+ and BSD`, but when I look at the nmap site, it refers to what I've got here as `licenses/rpm-nmap-ncat_gpl2-modified.txt`, which is GPLv2 with some 'clarifications and exceptions'.

**Comment by lgriffin** [comment]
I honestly don't know if we should be going down to RPM level for reviews on the license. Our only ask and task is to look after our own layer, so literally our software with our package.json view of it. I think it's a rabbit hole outside of that.

File: licenses/rpm-licenses.xml (hunk 1 above)
Comment (PCT-Pipelines) [reply licenses/rpm-licenses.xml]: @grdryn I think you can safely put what the License Field of the RPM metadata says

**Comment by PCT-Pipelines** [comment]
@grdryn @lgriffin From previous comments from Legal: "The licenses.xml file, for a given component (whether an RPM or other format) should only describe the licensing of bits inside that component. So if an RPM for qpid-proton* actually contained a bundled copy of cyrus-sasl-lib, say, then the license description should take that into account, otherwise no."
In this case, `memcached` and `nmap-ncat` I don't think bundle other rpms, so the only license of those 2 is enough, no additional build-time or run-time dependencies need to be analyzed.

**Comment by vibe13** [comment]
^^ That was me, sorry, was logged in as another user :-D

**Pull request closed without merging**